### PR TITLE
Migrate notification curl snippets

### PR DESCRIPTION
**Pull request by jefflinwood**

### CODE REVIEW

File: notifications/rest/credentials/create-apn-credential/create-apn-credential.json.curl [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (mcelicalderon) [inline notifications/rest/credentials/create-apn-credential/create-apn-credential.json.curl]: @jefflinwood Looks like this belongs to the output folder. Not a curl comand

Comment (jefflinwood) [inline notifications/rest/credentials/create-apn-credential/create-apn-credential.json.curl]: Yes - you are correct. Looks like I got these two mixed up. Will fix.

Comment (jefflinwood) [inline notifications/rest/credentials/create-apn-credential/create-apn-credential.json.curl]: These are now fixed. I also went through and took another look at each file to verify. Thanks!

**Comment by mcelicalderon** [comment]
@jefflinwood Just FWI the tests are failing as yesterday we introduced snippet testing. This shouldn't be a problem when this gets merged or if you rebase your branch. Also, because of this testing, we had to standardise the placeholders for account_sid and auth_token. So in order for a snippet to be testable, they should use ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX for account_sid and {{ auth_token }} for auth_token. Anyway, we are only testing for syntactic errors or compile errors and snippets are not tested by default. We are still working on the testing to improve functionality.

**Comment by jefflinwood** [comment]
Thanks @mcelicalderon - question for you then, do I need to go in and change all of the placeholders for account_sid and auth_token in this pull request?

**Comment by mcelicalderon** [comment]
@jefflinwood Well, I see you already changed most of them. This would probably be good to keep consistency across snippets, and only required if the snippets will be testable. I see the build is failing but that will be fixed once this gets merged into master. Are you still going to add more snippets or should I merge this one?

**Comment by jefflinwood** [comment]
Hi, can you merge this pull request, and then I can create the pages in the
CMS. I have the ruby snippets in a separate branch, but not with the new
auth tokens yet. I'll do a separate pull request for those.

Thanks!

**Pull request merged**